tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "compilerOptions": {
+        "skipLibCheck": true
+    }
```

Comment:
Do we really need this option?

Reply:
Unfortunately it is required for check-dts to work